compiler/rustc_attr/src/builtin.rs

```diff
@@ -405,11 +423,14 @@ where
                             };
                             if sym::unstable == meta_name {
                                 stab = Some((Stability { level, feature }, attr.span));
-                            } else {
+                            } else if sym::rustc_const_unstable == meta_name {
                                 const_stab = Some((
                                     ConstStability { level, feature, promotable: false },
                                     attr.span,
                                 ));
+                            } else {
```

Comment:
Could you change this to an `else if sym::rustc_default_body_unstable == meta_name` and add a panic in an `else` branch? As currently written, I think this is a footgun for anyone adding new stability attributes.

Reply:
Done